style/style.css

```diff
@@ -87,6 +65,181 @@ header a {
   background-image: url(/images/illustration-header-mobile-1.png);
   background-size: cover;
   background-position: center;
-  width: 190px;
-  height: 270px;
+  width: 147px;
+  height: 409px;
+}
+
+.portfolio-section {
+  width: 100%;
+}
+
+.content {
+  font-family: "Inter", sans-serif;
+  height: 50rem;
+  margin-top: 50rem;
+}
+
+.content h2 {
+  font-family: "Crete round", sans-serif;
+  font-weight: 400;
+  font-size: 40px;
+  color: #1c1a19;
+  text-align: center;
+}
+
+.content img {
+  width: 100%;
+  height: auto;
+}
+
+.content hr {
+  border-bottom: 2px solid #212121;
+  margin: 1.2rem auto 5rem;
+  width: 80%;
+}
+
+.content h3 {
+  margin: 1rem 0;
+  font-family: "Crete Round", sans-serif;
+  font-size: 32px;
+  font-weight: 400;
+  text-align: left;
+}
+
+.content p {
+  font-family: "Inter", sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-bottom: 1rem;
+}
+
+.content ul {
+  margin-bottom: 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.content ul li {
+  padding: 0.5rem;
+  border: 0.5px solid #8993a4;
+}
+
+.btn {
+  background-color: #ff6b00;
+  padding: 12px;
+  color: white;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+}
+
+.btn:hover {
+  background-color: #fff5e1;
+  border: 1px solid #ff6b00;
+  color: #ff6b00;
+}
+
+.project {
+  width: 100%;
+}
+
+.card {
+  width: 100%;
+  background: url("/images/ImgPlaceholder2.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-bottom: 3rem;
+}
+
+.card-content {
+  margin-left: 1.2rem;
+}
+
+.card h2 {
+  font-family: "Crete Round", sans-serif;
+  font-size: 36px;
+  font-weight: 400;
+  text-align: left;
+  margin: 5rem 0 1rem;
+  color: white;
+}
+
+.card p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 24px;
+  color: #fff;
+  margin-bottom: 1rem;
+}
+
+.card ul {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 1rem;
+}
+
+.card ul li {
+  color: white;
+  padding: 1rem;
+  background: rgba(255, 255, 255, 0.24);
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.card button {
+  width: 100%;
+  padding: 1rem;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: white;
+  background: #ff6b00;
+  margin-top: 1rem;
+}
+
+.about-section {
+  width: 100%;
+  height: 40rem;
+  background-image: url(/images/Frame.png);
+  background-size: 9rem;
+  background-repeat: no-repeat;
+  background-color: #1c1a19;
+  background-position-x: 231px;
+  background-position-y: 395px;
+}
```

Comment:
- [ ] Great job implementing this section and also using the correct background image. 👍 💯  I think there are still some improvements you need to make to the UI.
- - The background image is meant to be at the end of the screen, But in your project, for mobile screens bigger than `410px` the bg-image stays at the center, Please fix this as your portfolio is meant to match the Figma design.  😄 
- YOUR UI:
![Screenshot 2023-03-31 225814](https://user-images.githubusercontent.com/107632780/229241288-a88ef14a-5736-455c-9320-190860c55007.png)
- FIGMA DESIGN:
![Screenshot 2023-03-31 230004](https://user-images.githubusercontent.com/107632780/229241335-bf823772-c5ad-4f34-bac5-14b6036afb29.png)

Reply:
The background image has been fixed